Added description about default.xml (#4576)
magento/devdocs: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

To change the page wireframe, modify the [page layout] files; all other customizations are performed in the [page configuration] or [generic layout] files. 

In order to add a layout changes that will be available on every page, a module should use deafult.xml

For Example :

> Layout changes that is added to `app/design/frontend/OrangeCo/orange/Magento_Theme/layout/default.xml` will be loaded to all the available page.

Reviewer: We provide information on how to change layout globally but don't mention a possibility to make changes to particular handles. 
I would suggest adding another sentence about editing local handles. i.e. "To add layout changes to a specific page a module should use layout handle that corresponds the page's path. Example: changes that are added to `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` will be reflected on the product details page"

Reviewer: Also, if we are talking about a module, please replace `app/design/frontend/OrangeCo/orange/Magento_Theme/layout/default.xml` with something like `app/code/Vendor/Module/view/frontend/layout/default.xml`

Author: added the changes

To change the page wireframe, modify the [page layout] files; all other customizations are performed in the [page configuration] or [generic layout] files. 

> ## Layout handle

Reviewer 2: ```suggestion
## Manage layouts
```

> In order to add a layout changes that will be available on every page, a module should use `deafult.xml` layout.

Reviewer 2: ```suggestion
To make layout changes available on every page, modify the `default.xml` file.
```

> For example, layout changes that is added to `app/code/Vendor/Module/view/frontend/layout/default.xml` will be loaded on all pages.

Reviewer 2: ```suggestion
For example, layout changes added to `app/code/Vendor/Module/view/frontend/layout/default.xml` are loaded on all pages.
```

> To add layout changes to a specific page a module should use layout handle that corresponds the page's path. 

Reviewer 2: ```suggestion
To add layout changes to a specific page, use a layout file that corresponds to the page's path. 
```

> For example, the changes that are added to `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` will be reflected on the product details page.

Reviewer 2: ```suggestion
For example, changes to the `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` page are loaded on the product details page.
```
